### PR TITLE
fix issue-2279

### PR DESCRIPTION
**Pull request by sunilk747**
fix for - https://github.com/openshiftio/openshift.io/issues/2279

### CODE REVIEW

File: src/stack_license.py [hunk 1]
```diff
@@ -97,19 +97,20 @@ def compute_stack_license(self, payload):
         :param payload: Input list of package information
         :return: Detailed license analysis output
         """
+        # check input
+        if not payload or not payload['packages']:
+            output = dict()
```
Comment (sara-02) [inline src/stack_license.py]: We can simply do
```
output={'status': 'Failure'', 'message':''Input was invalid''}, as it is not going to change later.
```

File: src/stack_license.py [hunk 2]
```diff
@@ -97,19 +97,20 @@ def compute_stack_license(self, payload):
         :param payload: Input list of package information
         :return: Detailed license analysis output
         """
+        # check input
+        if not payload or not payload['packages']:
```
Comment (sara-02) [inline src/stack_license.py]: `payload['package]'` will throw an error if the package key is not found, and then having an if condition will be of no use. We can do `payload.get('package')` which will be more gracefully handled by the if condition, which is what we want.

Comment (sunilk747) [reply src/stack_license.py]: done

File: src/stack_license.py (hunk 1 above)
Comment (sunilk747) [reply src/stack_license.py]: done.

**Comment by sara-02** [comment]
@sunilk747 LGTM

**Pull request merged**